server/serialize.js

```diff
@@ -295,14 +299,14 @@ function objectHunt_(node, objectList) {
     if (typeof node === 'object') {  // Recurse.
       var names = Object.getOwnPropertyNames(node);
       for (var i = 0; i < names.length; i++) {
-        objectHunt_(node[names[i]], objectList);
+        Serializer.objectHunt_(node[names[i]], objectList);
       }
     }
   }
-}
+};
 
-if (typeof module !== 'undefined') { // Node.js
-  Interpreter = require('./interpreter.js');
-  exports.deserialize = deserialize;
-  exports.serialize = serialize;
+if (typeof module !== 'undefined') {  // Node.js
+  Interpreter = require('./');
```

Comment:
Now that there is actually a server 'main', we should probably change the entry in package.json to point to it, and continue to import `interpreter.js` explicitly.